grpc/src/main/java/io/stargate/grpc/payload/PayloadHandlers.java

```diff
@@ -20,18 +20,18 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import io.grpc.Status;
 import io.stargate.grpc.payload.cql.ValuesHandler;
-import io.stargate.proto.QueryOuterClass.Payload.Type;
```

Comment:
why dropping this import?

Reply:
Following usage from other parts of the code. Using `Payload.Type` in most places because `Type` is super generic.